src/sentry/tasks/post_process.py

```diff
@@ -763,6 +763,15 @@ def plugin_post_process_group(plugin_slug, event, **kwargs):
     )
 
 
+def get_post_process_queue(group_category: GroupCategory) -> str:
+    if group_category == GroupCategory.ERROR:
+        return "post_process_errors"
+    elif group_category == GroupCategory.PERFORMANCE:
+        return "post_process_performance"
+    else:
+        return "post_process_errors"  # use error queue as default
```

Comment:
Why is this needed? Shouldn't the category always be provided?

Reply:
not the cleanest code but generally would like to have a failsafe here. can remove the initial check for ERROR category for now

Reply:
The signature of this method indicates it's argument can only be a GroupCategory, if that's not true then the typing is probably not correct.

Reply:
right but there are potentially more error categories in the future so I wanted to be defensive in case it gets overlooked or we want to spin it up quickly